DuckDuckGo.Fluent.Plugin/HttpCalls.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Threading.Tasks;
+using static DuckDuckGo.Fluent.Plugin.JsonResult;
+
+namespace DuckDuckGo.Fluent.Plugin
+{
+    public static class HttpCalls
+    {
+        private const string UserAgentString = "Mozilla/5.0 (Windows NT 10.0; Win64; x64)";
+        private static readonly JsonSerializerOptions SerializerOptions = new() { PropertyNameCaseInsensitive = true };
+
+        public static async Task<DuckDuckGoApiResult> GetApiResult(string url)
+        {
+            using var httpClient = new HttpClient();
+            httpClient.Timeout = TimeSpan.FromSeconds(20);
```

Comment:
20 seconds? Isn't that too much

Reply:
I've removed the timeout as I felt the default value is good enough. I did not see any difference any so far w/o timeout.